indico/modules/events/api.py

```diff
@@ -341,6 +344,19 @@ def _build_session_api_data(self, session_):
         return [self._serialize_session_block(b, serialized_session, session_access_list, can_manage)
                 for b in session_.blocks]
 
+    @staticmethod
+    def serialize_reference(reference):
+        """Return the data for a reference"""
+        return {
+            'type': reference.reference_type.name,
+            'value': reference.value
+        }
+
+    @classmethod
+    def serialize_references(cls, references):
+        """Return the data for a list of references"""
+        return map(cls.serialize_reference, references)
```

Comment:
I could use
```python
return [{'type': reference.reference_type.name, 'value': reference.value} for reference in references]
```

It's a classmethod since I use it in the timetable serializer and that makes it clear from which serializer it comes from.

Should I also include the URL and URN?